scripts/utils/relayCompilerArguments.js

```diff
@@ -33,5 +33,5 @@ module.exports = [
   '--artifactDirectory',
   'src/__generated__',
 
-  process.env.PERSIST_QUERIES ? '--persist' : '',
+  process.env.PERSIST_QUERIES ? '--persist --persist-output' : '',
```

Comment:
the official relay persisted queries code didnt change much, except for this. passing both for now to support older projects using the forked version (we should make tickets to update those projects soon)